.travis.yml

```diff
@@ -49,3 +49,5 @@ env:
 before_install: ./Support/Testing/Travis/before-install.py
 install:        ./Support/Testing/Travis/install.py
 script:         ./Support/Testing/Travis/script.sh
+after_success:
+  - coveralls --exclude lib --exclude tests --gcov-options '\-lp'
```

Comment:
I think that's what happens yeah.

On Tue, Apr 5, 2016 at 2:46 PM, Francis Ricci <notifications@github.com>
wrote:

> In .travis.yml
> <https://github.com/facebook/ds2/pull/251#discussion_r58619203>:
>
> > @@ -49,3 +49,5 @@ env:
> >  before_install: ./Support/Testing/Travis/before-install.py
> >  install:        ./Support/Testing/Travis/install.py
> >  script:         ./Support/Testing/Travis/script.sh
> > +after_success:
> > +  - coveralls --exclude lib --exclude tests --gcov-options '\-lp'
>
> The tests passed. Maybe it's impossible to fail in an after_success block
> though?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/facebook/ds2/pull/251/files/4d169b3831571d17c54bc7a7cdfa399bb6ee7c44#r58619203>
>



-- 
Stephane Sezer